app/services/config_report_importer.rb

```diff
@@ -0,0 +1,9 @@
+class ConfigReportImporter < ReportImporter
```

Comment:
I'd say most of it should.. things like timings and log message storage helpers would be useful in the base class I'd say.